scripts/cargo-install-all.sh

```diff
@@ -49,6 +48,7 @@ BIN_CRATES=(
   cli
   bench-exchange
   bench-tps
+  genesis
```

Comment:
Can you add a big fat comment here and `BIN_CRATES+(genesis)`, to prevent the next person from coming along and put genesis back in alphabetical order

Reply:
Reference the issue in a comment too please, given how weird this is